internal/packer_registry/types.bucket.go

```diff
@@ -0,0 +1,288 @@
+package packer_registry
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-packer-service/preview/2021-04-30/models"
+	"github.com/hashicorp/packer/internal/packer_registry/env"
+	"google.golang.org/grpc/codes"
+)
+
+type Bucket struct {
+	Slug        string
+	Description string
+	Destination string
+	Labels      map[string]string
+	Config      ClientConfig
+	*Iteration
+	client *Client
+}
+
+// NewBucketWithIteration initializes a simple Bucket that can be used for tracking Packer
+// related build bits.
+func NewBucketWithIteration(opts IterationOptions) (*Bucket, error) {
+
+	b := Bucket{
+		Labels: map[string]string{
+			"ManagedBy": "Packer",
```

Comment:
```suggestion
			"CreatedBy": "Packer",
```
Might be more accurate ?